src/qml/controls/Theme.qml

```diff
@@ -0,0 +1,27 @@
+pragma Singleton
+import QtQuick 2.15
+
+QtObject {
+    property bool dark: false
+    property QtObject color: QtObject {
```

Comment:
2b2edac044bb68511558d91464608fb831b5816b

I think we could avoid so many ternaries:
```qml
readonly property ColorSet color: dark ? darkColorSet : lightColorSet

component ColorSet: QtObject {
    required property color background
    // ...
}

ColorSet {
    id: darkColorSet
    background: "black"
    // ...
}

ColorSet {
    id: lightColorSet
    background: "white"
    // ...
}
```

Reply:
Picked up in https://github.com/bitcoin-core/gui-qml/pull/168